docs/skopeo-list-tags.1.md

```diff
@@ -1,14 +1,14 @@
 % skopeo-list-tags(1)
 
 ## NAME
-skopeo\-list\-tags - List tags in the transport-specific image repository.
+skopeo\-list\-tags - List tags in the transport-specific image repository or a local docker-archive file.
```

Comment:
Non-blocking: How about something generic like “List image names in a transport-specific collection of images” ?

Reply:
this had done